Resources/Prototypes/_NF/Entities/Structures/Wallmounts/Signs/posters.yml

```
@@ -49,6 +49,26 @@
     sprite: _NF/Structures/Wallmounts/posters.rsi
     state: nf_poster5_legit
 
+- type: entity
+  parent: PosterBase
+  id: NFPosterLegitGunsPlease
+  name: "Guns Please!"
+  description: An adorable mothgirl in a cute hat asks that you please do your civic duty and give your guns to NFSD staff.
+  components:
+  - type: Sprite
+    sprite: _NF/Structures/Wallmounts/posters.rsi
+    state: nf_poster6_legit
+
+- type: entity
+  parent: PosterBase
+  id: NFPosterLegitHaveYouSeenThisEgg
+  name: "Have You Seen This Egg"
```

```suggestion
  name: "Have You Seen This Egg?"
```
Might read better with a question mark.